### PR TITLE
DE issue 1 to 8

### PR DESCRIPTION
**Pull request by Nanogamer7**
Issue 1 to 4 are newly translated and issue 5 to 8 have correction and addition to the previous version, including the new proofreader field (#219)

### CODE REVIEW

**Comment by Conor12345** [comment]
- [ ] Ensure changes made in PR #235 are maintained
- [ ] Ensure media and markdown has not been changed and are using latest versions

**Comment by Firerazer** [comment]
- [ ] Not the same names in translators and proofreaders. You must choose one or the other role

File: _posts/de/2020-09-11-ALTF4-4.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+## Das Entwickeln von Blaupausen mit Creative Mod und KirkMcDonald's Rechner <author>gamebuster800</author>
+
+Ich entwerfe meine Fabriken gerne in isolierter Umgebung. Wenn ich in meinem Spiel eine bestimmte Ressource, wie zum Beispiel Wissenschaftspaket benötige, dann pausiere ich mein „Survival“ Spiel und öffne einen Spielstand, in der [Creative Mod](https://mods.factorio.com/mod/creative-mod) (zu Deutsch: Kreativ Mod) aktiviert ist, um Blaupausen zu entwickeln, die meine Anforderungen erfüllen. Die Creative Mod fügt unendliche Quellen für Flüssigkeiten und Rohmaterialien, leicht konfigurierbare Cheats wie Zeitbeschleunigung, Abschalten von Beißern, unendliche Reichweite und Freischaltung aller Technologien hinzu. Die Mod ist die Inspiration für [Editor Extensions](https://mods.factorio.com/mod/EditorExtensions) (zu Deutsch: Editor Erweiterungen), welcher [letzte Woche](https://alt-f4.blog/de/ALTF4-3/#mod-spotlight-editor-extensions-raiguard) im Alt-F4 von Raiguard vorgestellt wurde.
```
Comment (Firerazer) [inline _posts/de/2020-09-11-ALTF4-4.md]: Alt-F4-3 link anchor

File: _posts/de/2020-09-18-ALTF4-5.md [hunk 2]
```diff
@@ -2,58 +2,59 @@
...
+Willkommen zur fünften Ausgabe von Alt-F4, und wir nehmen noch immer Fahrt auf! Viele Mitwirkende, Übersetzer und Technikfreaks helfen weiterhin fleißig, das Projekt fortzuführen, was einfach nur großartig ist. Wenn du am Spaß teilhaben möchtest, schau dir den *Beitragen*-Abschnitt am Ende der Ausgabe an. Zu den echten Neuigkeiten: Diese Woche argumentiert pocarski warum Landaufschüttung wieder weggrabbar sein sollte, Romner führt seinen Beitrag von [letzter Woche](https://alt-f4.blog/de/ALTF4-4/#mod-spotlight-realistic-fusion-power-romner) mit dem Vernichten von Beißern mit der Kraft der Fusion fort und Dr.MagPie nimmt uns zur Factorio Zone mit!
```
Comment (Firerazer) [inline _posts/de/2020-09-18-ALTF4-5.md]: Alt-F4-4 Anchor link

File: _posts/de/2020-09-18-ALTF4-5.md [hunk 3]
```diff
@@ -2,58 +2,59 @@
...
 
-Nachdem ich den [Realistic Fusion Power mod](https://mods.factorio.com/mod/RealisticFusionPower), in der Ausgabe von [letzter Woche](https://alt-f4.blog/ALTF4-4/#mod-spotlight-realistic-fusion-power-romner) vorgestellt, gemacht habe, fragte ich mich, ob es möglich ist, Fusionsenergie für etwas anderes als Stromversorgung verwenden könnte. Also, thermonukleare Bomben existieren bereits, und reine Fusionsbomben sind nur Theorie, aber ich hab sie trotzdem hinzugefügt. Ich glaube die gifs sprechen für sich:
+Nachdem ich die [Realistic Fusion Power Mod](https://mods.factorio.com/mod/RealisticFusionPower) (Realistische Fusionsenergie) gemacht habe (vorgestellt in der Ausgabe von [letzter Woche](https://alt-f4.blog/de/ALTF4-4/#mod-spotlight-realistic-fusion-power-romner)), fragte ich mich, ob es möglich ist, Fusionsenergie für etwas anderes als Stromversorgung zu verwenden. Also, thermonukleare Bomben existieren bereits, und reine Fusionsbomben sind nur Theorie, aber ich hab sie trotzdem hinzugefügt. Ich glaube die GIFs sprechen für sich:
```
Comment (Firerazer) [inline _posts/de/2020-09-18-ALTF4-5.md]: Alt-F4-4 Anchor link

File: _posts/de/2020-10-02-ALTF4-7.md [hunk 4]
```diff
@@ -1,87 +1,88 @@
...
+
+Viel Spaß beim Bauen!
+
+## Megabasis Denkweise <author>LoneWolf</author>
+
+In der [vierten Ausgabe von Alt-F4](https://alt-f4.blog/de/ALTF4-4/#designing-blueprints-with-creative-mod-and-kirkmcdonalds-calculator-gamebuster800) hat gamebuster800 kurz die Idee angesprochen, Erz direkt bei den Erzförderern zu schmelzen, statt an einem zentralen Ort. Der Grund dafür ist ziemlich unkompliziert und einfach zu verstehen: Erz stapelt sich auf 50, während Platten eine Stapelgröße von 100 erreichen. Daher bewegt man nur halb so viel, wenn man eine Ladung Erz transportiert, im Vergleich zu einer Ladung Platten. In kleinerem Maße, zum Beispiel wenn man seine Fabrik gerade erst begonnen hat, ist das kein Problem, aber wenn man in dem Bereich von zehntausenden Erz und Platten pro Minute kommt, kann so etwas einen großen Unterschied machen. Sogar mit Fließbändern ist Platten transportieren effizienter als Erz, aber mit Zügen ist der Unterschied überwältigend.
```
Comment (Firerazer) [inline _posts/de/2020-10-02-ALTF4-7.md]: Alt-F4-4 Anchor Link

File: _posts/de/2020-10-02-ALTF4-7.md [hunk 5]
```diff
@@ -1,87 +1,88 @@
...
+
+## Tritt morgen dem BCG MMO Event bei! <author>T-A-R</author>
+
+Letzte Woche wurde ich von Ki Tan von [Big Community Games](https://www.bigcommunitygames.com/Factorio-MMO) (zu Deutsch „Große Community Spiele“) mit aufregenden Neuigkeiten kontaktiert: Monatliche MMO Events, beginnend am ersten Samstag jeden Monats. Ja, du hast richtig gelesen: Es startet unmittelbar diesen Samstag!
+
+Wenn du mit dem Factorio Mehrspielermodus vertraut bist, kennst du vielleicht ein paar dieser Gastgeber schon. Neben der Veröffentlichungsparty „Willkommen zu Factorio“, in [Alt-F4 Ausgabe 3](https://alt-f4.blog/de/ALTF4-3/#dont-break-the-new-toys-please-t-a-r) erwähnt, sind sie für etliche große Events, inklusive dem unvergesslichen [Spiffing Brit Spiel](https://www.youtube.com/watch?v=2hgvIhMkgKU), verantwortlich.
```
Comment (Firerazer) [inline _posts/de/2020-10-02-ALTF4-7.md]: Alt-F4-3 Anchor Link

**Comment by Firerazer** [comment]
All localized internal links need translation accordingly to already published translation (#235)

File: _posts/de/2020-09-11-ALTF4-4.md (hunk 1 above)
Comment (Conor12345) [reply _posts/de/2020-09-11-ALTF4-4.md]: #mod-spotlight-editor-extensions-editor-erweiterungen-raiguard

File: _posts/de/2020-09-18-ALTF4-5.md (hunk 2 above)
Comment (Conor12345) [reply _posts/de/2020-09-18-ALTF4-5.md]: #mod-spotlight-realistic-fusion-power-realistische-fusionsenergie-romner

File: _posts/de/2020-09-18-ALTF4-5.md (hunk 3 above)
Comment (Conor12345) [reply _posts/de/2020-09-18-ALTF4-5.md]: #mod-spotlight-realistic-fusion-power-realistische-fusionsenergie-romner

File: _posts/de/2020-10-02-ALTF4-7.md (hunk 4 above)
Comment (Conor12345) [reply _posts/de/2020-10-02-ALTF4-7.md:50]: #designing-blueprints-with-creative-mod-and-kirkmcdonalds-calculator-gamebuster800

Comment (Conor12345) [reply _posts/de/2020-10-02-ALTF4-7.md]: #das-entwickeln-von-blaupausen-mit-creative-mod-und-kirkmcdonalds-rechner-gamebuster800

File: _posts/de/2020-10-02-ALTF4-7.md (hunk 5 above)
Comment (Conor12345) [reply _posts/de/2020-10-02-ALTF4-7.md]: #mache-bitte-nicht-die-neuen-spielzeuge-kaputt-t-a-r

**Pull request merged**